Sources/Apollo/InMemoryNormalizedCache.swift

```diff
@@ -32,9 +32,7 @@ public final class InMemoryNormalizedCache: NormalizedCache {
 
   public func clear(callbackQueue: DispatchQueue?,
```

Comment:
IMO we should make handler non optional here, otherwise its easy to miss that its asynchronous operation

Reply:
Eh, I can still see a case for people wanting it to fire off in a non-blocking way but not wanting to pass in a handler, I don't think it's worth it.